evm/spec/mpts.tex

```diff
@@ -12,15 +19,76 @@ \subsection{Internal memory format}
   \item A digest node is encoded as $(\texttt{MPT\_NODE\_HASH}, d)$, where $d$ is a Keccak256 digest.
 \end{enumerate}
 
+On the other hand the values or payloads are represented differently depending on the particular trie.
+
+\subsubsection{State trie}
+The state trie payload contains the account data. Each account is stored in 4 contiguous memory addresses containing
+\begin{enumerate}
+	\item the nonce,
+	\item the balance,
+	\item a pointer to the account's storage trie,
+	\item a hash of the account's code.
+\end{enumerate}
+The storage trie payload in turn is a single word.
+
+\subsubsection{Transaction Trie}
+The transaction trie nodes contain the length of the RLP encoded transaction, followed by the bytes of the RLP encoding of the transaction.
+
+\subsubsection{Receipt Trie}
+The payload of the recipts trie is a receipt. Each receipt is stored as
+\begin{enumerate}
+	\item the length in words of the payload,
+	\item the status,
```

Comment:
What are the possible statuses? Worth mentioning?

Reply:
For this part of document is enough to know that it fits in a 256 bits. Idk about the actual values